docs/bigcommerce-for-wordpress/extending-the-plugin/customization-guide.md

```diff
@@ -121,6 +132,7 @@ Generally, the WordPress plugin works with post IDs, not product IDs. The latter
 To retrieve additional information about the product in the browser, there is a REST API endpoint. Its primary purpose is supporting the product block interface in the WordPress admin, but it can be used anywhere to retrieve a small subset of the information about the product. The endpoint is `/wp-json/bigcommerce/v1/products`.

```

Comment:
There is a REST API endpoint to retrieve additional information about the product in the browser.  Its primary purpose is supporting the product block interface in the WordPress admin. Still, you can use it anywhere to retrieve a small subset of the product's information. The endpoint is `/wp-json/bigcommerce/v1/products`.